installation/routines/customize_options.sh

```diff
@@ -142,6 +142,58 @@ Would you like to overwrite your configuration? [Y/n]"
     fi
 }
 
+_option_spotify() {
+  # ENABLE_SPOTIFY
+  clear_c
+  print_c "---------------------- SPOTIFY ----------------------
+
+Installs an additional player, so that you can
+play music from spotify.
+
+Note: You need Spotify Premium to run this service!
+
+For more information see documentation:
+https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/spotify.md
```

Comment:
Just a thought, not a strong opinion: We never refer to the docs during the installation. It should be as easy as possible without the user to be required to go there. (I always think of the AutoHotspot as a good example there)

Reply:
Ok will refactor here